### PR TITLE
Changes to publish the artifacts on maven central

### PR DESCRIPTION
**Pull request by Sfurti-yb**
Changes to pom.xml
Removed Preparation of MultiHostConfiguration - With single host specified in the URL and loadBalanceHosts was true the application errored out with the error "Cannot prepare multihost config for single host specified"
Added URL test

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,22 +14,18 @@
   ~ limitations under the License.
   -->
 
-<project
-        xmlns="http://maven.apache.org/POM/4.0.0"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="
-                http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation=" http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (ashetkar) [inline pom.xml]: Let's try to keep our changes to minimum. If it's just formatting, please revert.

File: pom.xml [hunk 2]
```diff
@@ -63,9 +59,11 @@
     </licenses>
 
     <scm>
-        <connection>scm:git:https://github.com/pgjdbc/r2dbc-postgresql</connection>
-        <url>https://github.com/pgjdbc/r2dbc-postgresql</url>
-    </scm>
+        <connection>scm:git:git@github.com:yugabyte/r2dbc-postgresql.git</connection>
+        <developerConnection>scm:git:git@github.com:yugabyte/r2dbc-postgresql.git</developerConnection>
+        <url>https://github.com/yugabyte/r2dbc-postgresql</url>
+    <tag>r2dbc-postgresql-1.1.0-yb-1-alpha</tag>
```
Comment (ashetkar) [inline pom.xml]: nit: Indentation for the `<tag>`

File: pom.xml [hunk 3]
```diff
@@ -514,7 +518,7 @@
                                     <executable>java</executable>
                                     <arguments>
                                         <argument>-classpath</argument>
-                                        <classpath/>
+                                        <classpath />
```
Comment (ashetkar) [inline pom.xml]: nit: Extra whitespace

File: settings.xml [hunk 4]
```diff
@@ -10,3 +10,4 @@
 		</server>
 	</servers>
 </settings>
+
```
Comment (ashetkar) [inline settings.xml]: nit: extra newline

File: src/main/java/io/r2dbc/postgresql/PostgresqlConnectionFactoryProvider.java [hunk 5]
```diff
@@ -247,13 +247,27 @@ public final class PostgresqlConnectionFactoryProvider implements ConnectionFact
...
      * @since 1.0
      */
     public static final Option<TimeZone> TIME_ZONE = Option.valueOf("timeZone");
 
+    /**
+     * Enable ybServersRefreshInterval.
+     *
+     * @since 0.8.4
+     */
+    public static final Option<Integer> YB_SERVERS_REFRESH_INTERVAL = Option.valueOf("ybServersRefreshInterval");
```
Comment (ashetkar) [inline src/main/java/io/r2dbc/postgresql/PostgresqlConnectionFactoryProvider.java:269]: Have we handled `failed-host-reconnect-delay-secs` and `fallback-to-topology-keys-only` here?

Comment (Sfurti-yb) [reply src/main/java/io/r2dbc/postgresql/PostgresqlConnectionFactoryProvider.java:269]: We have handled `failed-host-reconnect-delay-secs` in the previous PR. `fallback-to-topology-keys-only` is not part of this release.

File: pom.xml (hunk 1 above)
Comment (Sfurti-yb) [reply pom.xml]: Done.

File: pom.xml (hunk 2 above)
Comment (Sfurti-yb) [reply pom.xml]: Done

File: pom.xml (hunk 3 above)
Comment (Sfurti-yb) [reply pom.xml]: Done.

File: settings.xml (hunk 4 above)
Comment (Sfurti-yb) [reply settings.xml]: Done.

**Review by ashetkar — APPROVED** [review]

**Pull request merged**